code/modules/antagonists/abductor/abductor.dm

```diff
@@ -73,12 +73,12 @@
 	owner.special_role = ROLE_ABDUCTOR
 	objectives += team.objectives
 	finalize_abductor()
-	ADD_TRAIT(owner, TRAIT_ABDUCTOR_TRAINING, ABDUCTOR_ANTAGONIST)
+	owner.add_traits(list(TRAIT_ABDUCTOR_TRAINING, TRAIT_CHUNKYFINGERS_IGNORE_BATON), ABDUCTOR_ANTAGONIST)
```

Comment:
why not just add this to the list of traits that the abductor species gets? why is the size of their fingers tied to their mind instead of their body?

Reply:
I thought it wouldn't cause edge-cases and inconsistencies where somehow non abductor antags species could use batons